src/components/v2/Layout/Sidebar/index.tsx

```diff
@@ -24,6 +25,8 @@ export const SidebarUi: React.FC = () => {
   const open = Boolean(anchorEl);
   const { t } = useTranslation();
   const styles = useStyles();
+  const { pathname } = useLocation();
+  const logoKey = pathname || useUID();
```

Comment:
You can't conditionally render a hook, it needs to render all the time.
In this case however it will never render, because `pathname` is never falsy.

Reply:
done the fix in other way.